### ISSUE TITLE
Xenia in a Flatpak

### ISSUE THREAD

**Issue by aliasbody** [xenia-project/xenia#2241]
For those not familiar with flatpak I'll let here an url quickly explaining what the goals are: https://docs.flatpak.org/en/latest/introduction.html

So I've built a file to create a flatpak version of xenia running the windows version with Wine (using the vulkan GPU). This is a barebone version just to know if the concept could work.

I'm creating this issue (don't know if a pull request or a contact via discord would be better to discuss this) in order to provide my help, if the xenia dev team sees it fit, to make this xenia an official flatpak distributable via flathub in order.

Here is the request in the flathub page (here I'm using a personal ID but if this could be oficially added it would be better and an official "jp.xenia" ID should be used.

Flathub pull request: https://github.com/flathub/flathub/pull/4835
Personal Github script location: https://github.com/aliasbody/flatpak-xenia

**Comment by tutacat** [comment]
What is the point of packing a flatpak for wine? wine-bottles or steam proton is more useful, and I don't see what the benefit of this is. You can already run wine, bottles, proton, and steam, through flatpak.

**Comment by n3thshan** [comment]
with steam os coming out to other platforms like the ROG Ally having a universal flatpak build will not only help those on different distros but also to those on **immutable** ones like bazzite and steam os itself which do not have write permissions on the root filesystem, thereby not allowing to install the app normally. This would thus be a step in the right direction for xenia devs since they are considering a native linux build.

**Comment by Twig6943** [comment]
This is for `canary` but https://github.com/streetpea/xenia-test/

**Comment by deadmeu** [comment]
> A good alternative to pushing a Flatpak is packaging it into an AppImage file; would help with not having to deal with Flatpak dependencies (multiple versions of FreeDesktop/GNOME/KDE Application Platforms) mucking up people's Linux installs.

This is not how Flatpaks work. The dependencies are not installed at the system level.  I believe the problem you are describing would occur with AppImages though.

**Comment by ttang4299** [comment]
> A good alternative to pushing a Flatpak is packaging it into an AppImage file

Nope, AppImages may be easy to install but they're more of a chore to keep updated. Meanwhile, there is a centralized way to update all Flatpaks from an app manager without needing to register for an account.
